pkg/controller/pxcbackup/controller.go

```diff
@@ -338,11 +338,9 @@ func (r *ReconcilePerconaXtraDBClusterBackup) runS3BackupFinalizer(cr *api.Perco
 
 			spl := strings.Split(cr.Status.Destination, "/")
 			backup := spl[len(spl)-1]
-			for _, bcp := range []string{backup + ".md5", backup + "sst_info", backup} {
-				err = retry.OnError(retry.DefaultBackoff, func(e error) bool { return true }, removeBackup(cr.Status.S3.Bucket, bcp, s3cli))
-				if err != nil {
-					break
-				}
+			err = retry.OnError(retry.DefaultBackoff, func(e error) bool { return true }, removeBackup(cr.Status.S3.Bucket, backup, s3cli))
```

Comment:
what about `.md5` file and `.sst_info/` dir?

Reply:
they will be deleted as well, because they have `backup` prefix in names